watson/cli.py

```diff
@@ -822,3 +822,111 @@ def sync(watson):
 
     watson.last_sync = arrow.utcnow()
     watson.save()
+
+
+@cli.command()
+@click.argument('frames', type=click.Path(exists=True))
+@click.argument('frames_with_conflict', type=click.Path(exists=True))
+@click.option('--no-dry-run', 'no_dry_run', is_flag=True,
+              help="If not specified, then merge will not be performed "
+              "and only merge statistics will be shown.")
+@click.pass_obj
+def merge(watson, frames, frames_with_conflict, no_dry_run):
+    """
+    Perform a merge of the frame file, with a conflicting frames file.
+
+    When storing the frames on a file hosting service, there is the
+    possibility that the frame file goes out-of-sync due to one or
+    more of the connected clients going offline. This can cause the
+    frames to diverge.
+
+    If the `--no-dry-run` command is not specified, only the statistics
+    of the merge will be shown, and the merge will not be performed.
+
+    The arguments `FRAMES` and `FRAMES_WITH_CONFLICT` are paths to the
+    `frames` and the conflicting `frames` file respectively.
+    """
+    original_frames, conflicting, merging = watson.merge_report(
+        frames, frames_with_conflict)
+
+    # find the length of the largest returned list, then get the number of
+    # digits of this length
+    dig = len(str(max(len(original_frames), len(merging), len(conflicting))))
+
+    click.echo("{:<{width}} frames will be left unchanged".format(
+        len(original_frames) - len(conflicting), width=dig))
+    click.echo("{:<{width}} frames will be merged".format(
+        len(merging), width=dig))
+    click.echo("{:<{width}} frames will need to be resolved".format(
+        len(conflicting), width=dig))
+
+    if not no_dry_run:
```

Comment:
I agree that this be better for a user's workflow. From the comments in issue #18, it looks like there is a desired option for the `--no-dry-run`. 

Would this prompt for the user to continue would be used only for the case where the user did not specify the `--no-dry-run` option?